shared/middleware/files_api.rb

```diff
@@ -395,10 +397,11 @@ def copy_file(endpoint, encrypted_channel_id, filename, source_filename)
   #
   # PUT /v3/(sources)/<channel-id>/<filename>?version=<version-id>
   # PUT /v3/(assets)/<channel-id>/<filename>
+  # PUT /v3/(libraries)/<channel-id>/<filename>
```

Comment:
Super-nit - can we update this comment style to match how we do the others in this file?

Reply:
Fixed!